eng/scripts/Create-APIView.ps1

```diff
@@ -158,8 +158,8 @@ function New-SwaggerAPIViewTokens {
     [string]$APIViewArtifactsDirectoryName
   )
 
-  $SourceCommitId = $(git rev-parse HEAD^)
-  $TargetCommitId = $(git rev-parse HEAD)
+  $SourceCommitId = $(git rev-parse HEAD^2)
```

Comment:
Can you help me understand why this is the case? When doing this for the common script https://github.com/Azure/azure-rest-api-specs/blob/main/eng/scripts/ChangedFiles-Functions.ps1#L1 HEAD should generally point to the merge commit and HEAD^ should be the changes from the PR.